index.js

```diff
@@ -91,9 +73,61 @@ server.get('/api/projects/:id' , (req , res) => {
     .catch(err => {
         res.status(500).json(err)
     })
-})    
+})      
+
```

Comment:
You're missing an important component of MVP: an endpoint that returns all of the actions linked to a given project (by the project id).